CONTRIBUTING.md

```diff
@@ -1,14 +1,95 @@
 # Contributors guide
 
-## Markdown style
-
-The specification documents follow the
-[Markdown Style Guide](http://www.cirosantilli.com/markdown-style-guide/). You
-can validate your changes against the guide using
-[remark](https://github.com/remarkjs/remark-lint) which works as a
-[standalone command line tool](https://github.com/remarkjs/remark/tree/master/packages/remark-cli)
-as well as
-[a plugin for various text editors](https://github.com/remarkjs/remark-lint#editor-integrations).
-You can also use [prettier](https://github.com/prettier/prettier) to
-automatically correct some of the style issuse that might be found in the
-proposed changes.
+# Contributing to the BIDS Specification
+
+:tada::balloon::cake: **Welcome to the BIDS Specification repository!** :cake::balloon::tada:
+
+:dizzy::hatched_chick::sunny: *We're so excited you're here and want to contribute.* :sunny::hatched_chick::dizzy:
+
```

Comment:
I don't think the BIDS Specification will want all the emojis from the starter kit. Maybe have a think about how this can be formalised a little while also still being friendly!

Reply:
yeah wasn't sure what we wanted to do about those, but maybe we can just stick to bold / italics then?

Reply:
I think just bold and italics is a good starting point ! We can add emojis back in later if we think it's missing something :)